inductiva/scenarios.py

```diff
@@ -48,32 +46,21 @@ def simulate(
         simulator: Simulator,
         output_dir: Optional[Path] = None,
         resource_pool_id: Optional[UUID] = None,
+        run_async: bool = False,
         **kwargs,
-    ) -> Path:
+    ) -> Union[Path, Task]:
         """Simulates the scenario synchronously."""
         self.validate_simulator(simulator)
-        output_dir = self._setup_output_dir(output_dir)
+
         with tempfile.TemporaryDirectory() as input_dir:
-            self.create_input_files(simulator, input_dir)
+            if not run_async:
```

Comment:
If it's a scenario and is synchronous we need to define the output name in a slightly different way (split the scenario name), so here we are doing it.

Reply:
My concern was that the `self.create_input_files(...)` should be done for both sync and async.

Reply:
I followed the code written previously, and there input files were created only when running synch simulations, I am not sure why.

Reply:
I think that was removed wrongly in a later PR, I have not noticed it locally because I was still running a previous version. Check here for a correct previous version: https://github.com/inductiva/inductiva/blob/109bd822e0115e6204700c343162d9f8491b9fc4/inductiva/scenarios.py

Previously, it was done with the self._setup_config(), now it is with the self.create_input_files.